### DISCUSSION TITLE
Colorblind themes should also be there for the dim theme

### DISCUSSION THREAD

**Discussion by DylanDevelops** [community/community#48643] [Accessibility]
### Select Topic Area

General

### Body

I am colorblind and appreciate all the themes being created for colorblind people but as I love the dim theme, as of now, there is no dim colorblind theme. If we could get that, it would be awesome!

**Comment by cehfisher** [comment]
Hello @DylanDevelops - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment]
Closing as this is a duplicate of [#27960](https://github.com/orgs/community/discussions/27960) opened by the same user.
